### PR TITLE
fix validation error

### PR DESCRIPTION
**Pull request by jjeongatl**
## Pull request description

_Provide description for the PR_

## Checklist
- [ ] I have successful end to end tests run (with & without domain)
- [ ] I have added unit tests (if applicable)
- [ ] I have user documentation (if applicable)

### CODE REVIEW

File: modules/AWS/eks/nodegroup_launch_template/locals.tf [hunk 1]
```diff
@@ -4,34 +4,34 @@ locals {
   # If the region is found in any of the supported regions lists, it will be used as kinesis region in osquery flags.
   # Otherwise either the default european or non european region and role are used
 
-  aws_sts_arn_role_eu       = "${var.kinesis_log_producers_role_arns["eu"]}"
-  aws_sts_arn_role_non_eu   = "${var.kinesis_log_producers_role_arns["non-eu"]}"
+  aws_sts_arn_role_eu     = var.kinesis_log_producers_role_arns["eu"]
+  aws_sts_arn_role_non_eu = var.kinesis_log_producers_role_arns["non-eu"]
```
Comment (bconroyA) [inline modules/AWS/eks/nodegroup_launch_template/locals.tf:8]: Just for my own learning, why was this removed?

**Review by bianchi2 — APPROVED** [review]

File: modules/AWS/eks/nodegroup_launch_template/locals.tf (hunk 1 above)
Comment (bianchi2) [reply modules/AWS/eks/nodegroup_launch_template/locals.tf:8]: fmt removed redundant ${} as it's typically used for interpolation which isn't the case here.
